routers/wanaku-router/src/main/java/ai/wanaku/routers/proxies/tools/InvokerProxy.java

```diff
@@ -72,12 +74,43 @@ private static ToolInvokeReply invokeRemotely(
         Map<String, String> serviceConfigurations = Configurations.toStringMap(configurations);
         Map<String, String> argumentsMap = CollectionsHelper.toStringStringMap(toolArguments.args());
 
+        Map<String, ToolReference.Property> inputSchema =toolReference
+                .getInputSchema()
+                .getProperties();
+
+        Map<String,String> serviceEndpointConfiguration = inputSchema.entrySet()
+                .stream()
+                .filter(entry -> {
+                    ToolReference.Property property = entry.getValue();
+                    return property !=null &&
+                    property.getTarget().equals("configuration") &&
+                    property.getScope().equals("service-endpoint");
```

Comment:
Let's use a constant for these and for the `header` and the `service` strings below. It makes it easier for us to document this.